wordpress-mobile/WordPress-iOS
# Reader: Remove Reader filter bar UI

Closes #22206 

> [!NOTE]
> This merges into `feature/reader-navigation-filter`.

## Description

Removes the existing filter UI from the Reader. I left the functions which apply the filter and reset the filter, since they'll likely be used in the future.

## Screenshot

![Simulator Screenshot - iPhone SE (3rd generation) - 2023-12-11 at 13 52 30](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/1ff6b17a-f0e8-4925-9dbd-b27ced08eec4)

## Testing

To test:
- Launch Jetpack and login
- Navigate to the "Following" tab in the Reader
- 🔎 **Verify** the filter bar no longer appears
- Navigate to the "Automattic" filter
- 🔎 **Verify** the filter bar no longer appears

## Regression Notes
1. Potential unintended areas of impact
Should be none

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
